test/issues/0217.js

```diff
@@ -5,7 +5,7 @@ var assert = require('assert');
 var yaml   = require('../../');
 
 
-test('Javascript functions are not folded', function () {
+test.skip('Javascript functions are not folded', function () {
```

Comment:
I mean, tests were added to check that fns & regexes are not broken. I don't mean tests should stay intact, but those should continue to work. There are no need to disable those cimpletely, for historic reasons.